cluster/validation.go

```diff
@@ -344,6 +344,11 @@ func validateDNSImages(c *Cluster) error {
 			return fmt.Errorf("coredns autoscaler image is not populated")
 		}
 	}
+	if c.DNS.Nodelocal != nil {
+		if len(c.SystemImages.Nodelocal) == 0 {
```

Comment:
do this in one line?

Reply:
This was to keep the style with the other systemimages checks, but as this is only one separate condition I merged them